src/SFML/Window/SensorManager.hpp

```diff
@@ -30,6 +30,8 @@
 #include <SFML/Window/Sensor.hpp>
 #include <SFML/Window/SensorImpl.hpp>
 
+#include <SFML/System/EnumArray.hpp>
+
```

Comment:
Extra empty line.

Reply:
This is preserving the two empty newlines between the last head and `namespace sf::priv`.

Reply:
Ah, right!